### PR TITLE
dlangspec.pdf: table of contents

### PR DESCRIPTION
**Pull request by aG0aep6G**
before/after:
iconscreenshot1

### CODE REVIEW

**Comment by CyberShadow** [comment]
Nice. I thought the TOC was broken because of some Latex invocation weirdness (you have to run it twice or something)

File: latex.ddoc [hunk 1]
```diff
@@ -3,6 +3,7 @@ _=Macros for LaTeX generation. The following section is ordered alphabeticaly.
 _=*****************************************************************************
 
 A=\ref{$1}{$+}
+AHTTP=\href{http://$1}{$+}
```
Comment (CyberShadow) [inline latex.ddoc]: Question, why do we need both `AHTTP` and `WEB`?

Comment (aG0aep6G) [inline latex.ddoc]: I have no idea. There's also `HTTP`. The differences (if any) between them are not clear to me at all.

**Comment by CyberShadow** [comment]
Thanks!

**Pull request merged**